example/scp.c

```diff
@@ -43,7 +43,11 @@ int main(int argc, char *argv[])
     const char *scppath="/tmp/TEST";
     libssh2_struct_stat fileinfo;
     int rc;
+#if defined(WIN32) & defined(HAVE_STRTOI64)
```

Comment:
Ok, I tested this on win32 wint libssh2_struct_stat_size instead of off_t and it works just fine.  I didn't get a warning with off_t on win32 but it would wrap and go negative and would never recover in some cases, which is worse.